doc/releases/changelog-dev.md

```diff
@@ -33,42 +33,116 @@
 
 <h4>Access an extended arsenal of quantum algorithms 🏹</h4>
 
-
-* The `FABLE` template is added for efficient block encoding of matrices. Users can now call FABLE to efficiently construct circuits according to a user-set approximation level. 
+* The Fast Approximate BLock-Encodings (FABLE) algorithm 
+  (https://arxiv.org/abs/2205.00081) is now accessible via the `qml.FABLE` 
+  template.
   [(#5107)](https://github.com/PennyLaneAI/pennylane/pull/5107)
 
-* Create the `qml.Reflection` operator, useful for amplitude amplification and its variants.
-  [(#5159)](https://github.com/PennyLaneAI/pennylane/pull/5159)
+  The usage of `qml.FABLE` is similar to `qml.BlockEncode` but provides a more
+  efficient circuit construction at the cost of a user-defined approximation 
+  level, `tol`. The number of wires that `qml.FABLE` operates on is `2*n + 1`, 
+  where `n` defines the dimension of the $2^n \times 2^n$ matrix that we want to 
+  block-encode.
 
   ```python
-  @qml.prod
-  def generator(wires):
-        qml.Hadamard(wires=wires)
-
-  U = generator(wires=0)
-
+  A = np.random.random((2, 2)) # n = 1, 2*n + 1 = 3
   dev = qml.device('default.qubit')
+
   @qml.qnode(dev)
   def circuit():
+      qml.FABLE(A, tol = 0.1, wires=range(3))
+      return qml.state()
+  ``` 
+
+  ```pycon
+  >>> circuit()
+  tensor([0.29906437+0.j, 0.20579772+0.j, 0.29906437+0.j, 0.20579772+0.j,
+          0.40070002+0.j, 0.45568333+0.j, 0.40070002+0.j, 0.45568333+0.j], requires_grad=True)
+  ```
 
-        # Initialize to the state |1>
-        qml.PauliX(wires=0)
+* Reflecting about a given quantum state is now available via `qml.Reflection`.
+  This operation is very useful in, say, the amplitude amplification algorithm.
+  [(#5159)](https://github.com/PennyLaneAI/pennylane/pull/5159)
 
-        # Apply the reflection
-        qml.Reflection(U)
+  `qml.Reflection` works by providing an operation, $U$, that *prepares* the 
+  desired state, $\vert \psi \rangle$, that we want to reflect about. In other 
+  words, $U$ is such that $U \vert 0 \rangle = \vert \psi \rangle$. In 
+  PennyLane, $U$ must be an `Operator`.
+  
+  For example, if we want to reflect about 
+  $\vert \psi \rangle = \vert + \rangle$, then $U = H$:
```

Comment:
Can we tailor the mathematical snippets to use
```
:math:`...`
```
This will show up in the [built docs](https://xanaduai-pennylane--5574.com.readthedocs.build/en/5574/development/release_notes.html#release-0-36-0-dev-development-release). Unfortunately, that makes it not show up in the [GitHub release notes](https://github.com/PennyLaneAI/pennylane/releases/tag/v0.35.0), but we have the option of editing those.

Reply:
Yep! I just forgot is all 🥴